src/pages/guides/sellers/app-version.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Specify the version
+---
+
+# Specify the version
+
+After you [create the app entry](app-create.md), the next step is to specify and submit a new version. Extensions can have multiple versions, including the first version submitted.
+
+For existing, approved extensions, follow these instructions to add new versions as available and released.
+
+1. Log in to the Marketplace Developer Portal.
+
+1. Click **Extensions**, and then choose an extension from the list.
```

Comment:
If this section is really for **apps**, then the developer would click `Apps` (as opposed to `Extensions`).